.clang-format

```diff
@@ -0,0 +1,94 @@
+# OSARA: Open Source Accessibility for the REAPER Application
+# .clang-format
+# Author: James Teh <jamie@jantrid.net>
+# Copyright 2023 Leonard de Ruijter
+# License: GNU General Public License version 2.0
+
+# Style based on Google's C++ style guide.
+BasedOnStyle: Google
+# Offset for access modifiers (public, protected, private).
+AccessModifierOffset: 0
+# Align code after an open bracket with block indentation.
+AlignAfterOpenBracket: DontAlign
```

Comment:
Hmm, I don't love what this does to some data structures. I'm pondering switching back to BlockIndent here. I do feel BlockIndent wastes lines in some cases - a function call that could be 2 lines ends up being 4 - but at least it's consistent across the board. I wonder if this would also fix the weirdness in uia.cpp. I'll have to try it.

Reply:
> I wonder if this would also fix the weirdness in uia.cpp.

I'm afraid not. This might just be a weird bug in clang-format itself.